docusaurus/docs/publish-a-plugin/sign-a-plugin.md

```diff
@@ -1,74 +1,88 @@
-| ---------------- | ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Private          | No;Free of charge                           | You can create and sign a Private plugin for any technology at no charge.Private plugins are intended for use on your own installation of Grafana. They may not be distributed to the Grafana community, and they are not published in the Grafana plugin catalog.                                                 |
-| Community        | No;Free of charge                           | You can create, sign, and distribute plugins at no charge, provided that all dependent technologies are open source and not for profit.Community plugins are published in the official Grafana catalog, and are available to the entire Grafana community.                                                         |
-| Commercial       | Yes;Commercial plugin subscription required | You can create, sign, and distribute plugins with dependent technologies that are closed source or commercially backed. To do so, enter into a Commercial plugin subscription with Grafana Labs.Commercial plugins are published on the Grafana plugin catalog, and are available to the entire Grafana community. |
+| **Plugin Level** | **Paid Subscription Required?**                 | **Description**                                                                                                                                                                                                                                                                                                                  |
+| ---------------- | ----------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Private          | No;<br></br>Free of charge                           | <p>You can create and sign a Private plugin for any technology at no charge.</p><p>Private plugins are intended for use on your own installation of Grafana. They may not be distributed to the Grafana community, and they are not published in the Grafana plugin catalog.</p>                                                 |
```

Comment:
Please update this table with the exact copy from https://grafana.com/legal/plugins/ we should ensure they're identical prior to publishing.

Reply:
Done.